src/components/Article.jsx

```diff
@@ -19,14 +20,23 @@ export default function Article() {
 
   return (
     <div>
-      <img src={article.article_img_url} alt={article.title} />
+      <img
+        className="article-img2"
+        src={article.article_img_url}
+        alt={article.title}
+      />
       <h2>{article.title}</h2>
       <p>Author: {article.author}</p>
       <p>Topic: {article.topic}</p>
       <p>Created at: {new Date(article.created_at).toLocaleString()}</p>
       <p>{article.body}</p>
+      <p>{article.body}</p>{" "}
+      <Votes
+        article_id={article_id}
+        initialVotes={article.votes}
+        onVoteChange={(newVotes) => setArticle({ ...article, votes: newVotes })}
```

Comment:
I recommend removing this because updating this is pointless as you are going to pass down votes to the Vote component but that is not necessary as the Vote component has its own state of votes that it is keeping and its updating the votes in the backend.